### PR TITLE
ad loader in group tab

### PR DESCRIPTION
**Pull request by veolys-nanne**
[OO-FEATURE] add loader in group tab view
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1467

### CODE REVIEW

File: MediaAdminBundle/Resources/public/coffee/groupTree/FormMediaFolderCollectionView.coffee [hunk 1]
```diff
@@ -1,68 +1,15 @@
 ###*
- * @namespace OpenOrchestra:GroupTree
-###
-window.OpenOrchestra or= {}
-window.OpenOrchestra.GroupTree or= {}
-
-###*
  * @class FormMediaFolderCollectionView
```
Comment (alavieille) [inline MediaAdminBundle/Resources/public/coffee/groupTree/FormMediaFolderCollectionView.coffee:8]: Why remove comment of namespace and why remove initialize of namespace ?

File: MediaAdminBundle/Resources/public/coffee/groupTree/FormMediaFolderCollectionView.coffee [hunk 2]
```diff
@@ -1,68 +1,15 @@
...
-    folderGroupRoleData = []
-    folderGroupRoleData.push({'model_id': folderId, 'type':'folder', 'access_type': value, 'name': name})
-    $.ajax
-      url: @options.group.links._self_edit
-      method: 'POST'
-      data: JSON.stringify(
-        model_roles: folderGroupRoleData
-      )
+class OpenOrchestra.GroupTree.FormMediaFolderCollectionView extends OpenOrchestra.GroupTree.AbstractFormCollectionView
+  getElement: ->
```
Comment (alavieille) [inline MediaAdminBundle/Resources/public/coffee/groupTree/FormMediaFolderCollectionView.coffee:14]: can you add comment for this methods

**Comment by veolys-nanne** [comment]
done @itkg-alavieille

**Comment by ngilain** [comment]
:+1:

**Pull request merged**